census_example.ipynb

```diff
@@ -174,6 +174,15 @@
     "y_test = train[target].values[test_indices]"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "max_epochs = 1000 if not os.getenv(\"CI\", False) else 20"
```

Comment:
Wouldn't 5 or 10 epochs be enough here? Just so the test can be faster :)

Reply:
I think 5 or 10 would actually be enough for the tests, nothing specific happens between 5th and 20th epoch